api/PublicKeyCredential.json

```diff
@@ -60,7 +60,7 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": false
+              "version_added": "120"
```

Comment:
FWIW this is correct - added in https://bugzilla.mozilla.org/show_bug.cgi?id=1810851